### ISSUE TITLE
When you don't need a tuple

### ISSUE THREAD

**Issue by martinrojas** [harryhope/hooked-on-redux#3]
First off congratulations on such an amazing and clean take on hooks use in redux. 

I think that for this library (or documentation) is missing 2 use cases for it to be used in a real-world project.

1. Call an action that sets a value on the store, but it's not needed in that component. 
2. get a specific value without needing a setter function

There are other ways of doing this in Redux, but I was wondering about the best way to solve this problem in a "best-practice" with this library

**Comment by harryhope** [comment]
Thanks @martinrojas , I appreciate your kind words and feedback.

On your points:

1) I've been thinking this would be useful as well. Something like `setValue('path.to.value', value)`. Is that along the lines of what you were thinking?

2) Correct me if I'm not understanding you, but can't you just write `const [value] = useHookedOnState('path.to.value', null)`?

**Comment by martinrojas** [comment]
1. That is something that I had in mind when I wrote this up. However, as I looked into the other code in the repo I saw the following implementation in your test `const [,updateAll] = useHookedOnState('app', {})` I am testing this and seems to be accomplishing it. 
2. Now it seems like the same use as the one above and it lines up with your solution. I have not tested that yet. 

My concern is if this causes some sort of memory leak. I will continue to do research. If there is no solution then it may just be a case of updating documentain.
